code/modules/antagonists/clock_cult/mobs/cogscarab.dm

```diff
@@ -0,0 +1,60 @@
+#define CLOCKDRONE	"drone_clock"
+
+//====Cogscarab====
+
+/mob/living/simple_animal/drone/cogscarab
```

Comment:
Add: `initial_language_holder = /datum/language_holder/clockmob` somewhere under this

Reply:
and may I also suggest sneaking these two in somewhere under servant_servant_of_ratvar.dm:
```
grant_language(/datum/language/ratvar, TRUE, TRUE, LANGUAGE_CULTIST)

remove_language(/datum/language/ratvar, TRUE, TRUE, LANGUAGE_CULTIST)
```